inc/saiswitch.h

```diff
@@ -406,6 +406,140 @@ typedef enum _sai_tunnel_decap_ecn_mode_t
 
 } sai_tunnel_decap_ecn_mode_t;
 
+/**
+ * @brief SAI switch soft error recovery error type.
+ */
+typedef enum _sai_switch_soft_error_type_t
+{
+    /**
+     * @brief Unknown error type
+     */
+    SAI_SWITCH_SOFT_ERROR_TYPE_UNKNOWN = 0,
+
+    /**
+     * @brief Parity error
+     */
+    SAI_SWITCH_SOFT_ERROR_TYPE_PARITY = 1,
+
+    /**
+     * @brief ECC single bit error
+     */
+    SAI_SWITCH_SOFT_ERROR_TYPE_ECC_SINGLE_BIT = 2,
+
+    /**
+     * @brief ECC double bit error
+     */
+    SAI_SWITCH_SOFT_ERROR_TYPE_ECC_DOUBLE_BIT = 3,
+
+    /**
```

Comment:
Is this meant to be "all other soft error types"? If so, how does it differ from UNKNOWN? (if not, what is it?)

Reply:
checkout here: https://github.com/opencomputeproject/SAI/blob/master/inc/saitypes.h#L1446